layouts/partials/menu.html

```diff
@@ -26,6 +26,21 @@
         {{ end }}
       {{ end }}
     {{ end }}
+
+    {{ if $.Site.Params.showLanguageSelector }}
+    <div class="spacer"></div>
+    <div class="language-selector">
+
+      <div class="language-selector-actual">
+          <li class="list-inline-item">{{ .Language.LanguageName }} ▾</li>
+      </div>
+      <div class="language-selector-more hidden">
```

Comment:
```suggestion
      <ul class="language-selector__more hidden">
```

Please, use semantic markup and BEM class style ;-)